src/wallet/rpcwallet.cpp

```diff
@@ -4082,6 +4163,8 @@ UniValue walletcreatepsbt(const JSONRPCRequest& request)
                             "using fundrawtransaction.\n"
                             "\nArguments:\n"
                             "1. \"hexstring\"            (string, required) The hex string of the raw transaction\n"
+                            "2. \"include_output_info\"  (boolean, optional, default=false) If true, returns the PSBT with the redeem scripts, witness\n"
```

Comment:
you need to edit number of allowed args, and include the optional one in parens above.

Reply:
you also need to add it to the list of arguments that will be parsed as json rather than strings in `rpc/client.cpp`

Reply:
Dang, I knew I forgot something. Fixed.